cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt

```diff
@@ -338,7 +338,7 @@ script "js_exclude_tags", type: "javascript" do
         resourceTypeSplit = ipAddress.resourceType.split("/")
         service = resourceTypeSplit[0]
 
-        if(properties.ipConfiguration == null){
+        if(properties.ipConfiguration == null && properties.natGateway == null){
```

Comment:
Normalize the tabs here to make the code more readable. If the second if statement is nested within the first, it should be tabbed in further.